src/apis/impl/AuthenticationApiImpl.ts

```diff
@@ -187,14 +204,14 @@ export class AuthenticationApiImpl implements AuthenticationApi {
     try {
       const user = await api.baseApi.userApi.getCurrentUser();
       if (user == null) {
-        throw Error(`Your validation code ${validationCode} is expired - Couldn't login new user`);
+        throw this.errorHandler.createErrorWithMessage(`Your validation code ${validationCode} is expired - Couldn't login new user`);
       }
 
       const [providedToken, keyPair] = tokenAndKeyPairProvider(user.groupId!, user.id!) ?? [undefined, undefined];
 
       const token = providedToken ?? await api.userApi.createToken(user.id!, 3600 * 24 * 365 * 10);
       if (token == null) {
-        throw Error(`Your validation code ${validationCode} is expired - Couldn't create new secured token`);
+        throw this.errorHandler.createErrorWithMessage(`Your validation code ${validationCode} is expired - Couldn't create new secured token`);
```

Comment:
`Your validation code ${validationCode} expired. Start a new authentication process for your user`